default/methods/project/project_new.py

```diff
@@ -78,14 +78,7 @@ def project_new_api():
             member_created=user.member
         )
 
-        schema = LabelSchema.new(
```

Comment:
Hey Anthony! I'm curious why the default schema creation was removed here?

Where is the label schema is created now when a new project is created?